lib/client-assets.php

```diff
@@ -629,10 +629,7 @@ function gutenberg_extend_block_editor_styles( $settings ) {
  * @return array Block Pattern Array.
  */
 function gutenberg_load_block_pattern( $name ) {
-	return json_decode(
-		file_get_contents( __DIR__ . '/patterns/' . $name . '.json' ),
-		true
-	);
+	return require( __DIR__ . '/patterns/' . $name . '.php' );
```

Comment:
Do we need to keep support for both for a bit?

Reply:
No we  don't, since this  was a function that only loaded the fixed set of patterns in Gutenberg and all of these have been migrated. There's no possibility this was used by third-party users.